{
  "uid" : "d8b73062e72f8531",
  "name" : "test_login",
  "fullName" : "Selenium-Tests.test_login_improper.TestLogin#test_login",
  "historyId" : "e9fbbaa2fbdece056b74a1fba76c5170",
  "time" : {
    "start" : 1629125247224,
    "stop" : 1629125250924,
    "duration" : 3700
  },
  "status" : "broken",
  "statusMessage" : "NameError: name 'driver' is not defined",
  "statusTrace" : "self = <test_login_improper.TestLogin object at 0x7f1bff7827c0>\n\n        def test_login(self):\n            self.driver.get(URL)\n            time.sleep(2)\n            sign_in = self.driver.find_element_by_xpath('//ul/li[3]/a')\n            sign_in.click()\n    \n    # Üresen elküldött login form\n    \n            email_field = self.driver.find_element_by_xpath('//form/fieldset[1]/input[1]')\n            email_field.click()\n            email_field.send_keys(\"\")\n    \n            password_field = self.driver.find_element_by_xpath('//form/fieldset[2]/input[1]')\n            password_field.click()\n            password_field.send_keys(\"\")\n    \n            sign_in_btn = self.driver.find_element_by_xpath('//*[@id=\"app\"]/div/div/div/div/form/button')\n            sign_in_btn.click()\n    \n            element = self.driver.find_element(By.CSS_SELECTOR, \".btn\")\n            actions = ActionChains(self.driver)\n            actions.move_to_element(element).perform()\n>           driver.find_element_by_xpath('/html/body/div[2]/div/div[4]/div/button').click()\nE           NameError: name 'driver' is not defined\n\nSelenium-Tests/test_login_improper.py:50: NameError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "xunit_setup_method_fixture_TestLogin",
    "time" : {
      "start" : 1629125247007,
      "stop" : 1629125247223,
      "duration" : 216
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "NameError: name 'driver' is not defined",
    "statusTrace" : "self = <test_login_improper.TestLogin object at 0x7f1bff7827c0>\n\n        def test_login(self):\n            self.driver.get(URL)\n            time.sleep(2)\n            sign_in = self.driver.find_element_by_xpath('//ul/li[3]/a')\n            sign_in.click()\n    \n    # Üresen elküldött login form\n    \n            email_field = self.driver.find_element_by_xpath('//form/fieldset[1]/input[1]')\n            email_field.click()\n            email_field.send_keys(\"\")\n    \n            password_field = self.driver.find_element_by_xpath('//form/fieldset[2]/input[1]')\n            password_field.click()\n            password_field.send_keys(\"\")\n    \n            sign_in_btn = self.driver.find_element_by_xpath('//*[@id=\"app\"]/div/div/div/div/form/button')\n            sign_in_btn.click()\n    \n            element = self.driver.find_element(By.CSS_SELECTOR, \".btn\")\n            actions = ActionChains(self.driver)\n            actions.move_to_element(element).perform()\n>           driver.find_element_by_xpath('/html/body/div[2]/div/div[4]/div/button').click()\nE           NameError: name 'driver' is not defined\n\nSelenium-Tests/test_login_improper.py:50: NameError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "9f02033b921cbde5",
      "name" : "log",
      "source" : "9f02033b921cbde5.txt",
      "type" : "text/plain",
      "size" : 344
    }, {
      "uid" : "9cf1fb4226d40b7f",
      "name" : "stderr",
      "source" : "9cf1fb4226d40b7f.txt",
      "type" : "text/plain",
      "size" : 215
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "xunit_setup_method_fixture_TestLogin::0",
    "time" : {
      "start" : 1629125250967,
      "stop" : 1629125251025,
      "duration" : 58
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "Selenium-Tests"
  }, {
    "name" : "suite",
    "value" : "test_login_improper"
  }, {
    "name" : "subSuite",
    "value" : "TestLogin"
  }, {
    "name" : "host",
    "value" : "fv-az83-229"
  }, {
    "name" : "thread",
    "value" : "3682-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "Selenium-Tests.test_login_improper"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 2,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "cf37d7dcc5a8a36f",
        "reportUrl" : "https://RankyLea.github.io/conduit/27//#testresult/cf37d7dcc5a8a36f",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)",
        "time" : {
          "start" : 1629124719330,
          "stop" : 1629124719330,
          "duration" : 0
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "d8b73062e72f8531.json",
  "parameterValues" : [ ]
}